build.include

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Changing `744` to `u+x`. We do not know when someone might use the wrong editor. I think Windows might have this issue if the files are stored on NTFS filesystem. Think it wise to keep this.